types/react-native/test/index.tsx

```diff
@@ -1059,6 +1059,13 @@ export class ImageTest extends React.Component {
             (width, height) => console.log(width, height),
             error => console.error(error),
         );
+        Image.prefetch(uri).then((success) => {
+            if (success) {
```

Comment:
Instead of this code, perhaps a single `$ExpectType` assertion here on the `success` variable would be a better test.